legend-sdlc-language-pure-compiler/src/main/java/org/finos/legend/sdlc/language/pure/compiler/toPureGraph/PureModelBuilder.java

```diff
@@ -135,6 +136,12 @@ public PureModelBuilder withEntitiesIfPossible(Entity... entities)
         return this;
     }
 
+    public PureModelBuilder withSDLC(SDLC sdlc)
```

Comment:
Should we also add builder methods for setting the protocol?

Reply:
Added this